packages/gatsby/src/utils/handle-flags.ts

```diff
@@ -143,7 +145,7 @@ const handleFlags = (
     if (optedInFlags.size > 0) {
       message += `\n\n`
       message += `We're shipping new features! For final testing, we're rolling them out first to a small % of Gatsby users
-and your site was automatically choosen as one of them. With your help, we'll then release them to everyone in the next minor release.
+and your site was automatically chosen as one of them. With your help, we'll then release them to everyone in the next minor release.
```

Comment:
randomly spoted typo

Reply:
You mean randomly _spotted_ typo :trollface:

Reply:
hahaha, oh the irony xD